tests/test_sklearn_one_vs_rest_classifier_converter.py

```diff
@@ -127,6 +130,18 @@ def test_ovr_classification_decision_function_binary(self):
             "<= StrictVersion('0.2.1')",
             methods=['predict', 'decision_function_binary'],
         )
+        if StrictVersion(ort_version) < StrictVersion("1.0.0"):
+            return
+        options = {id(model): {'raw_scores': True, 'zipmap': False}}
+        model_onnx = convert_sklearn(
+            model, "ovr classification",
+            [("input", FloatTensorType([None, X.shape[1]]))],
+            options=options, target_opset=TARGET_OPSET)
+        sess = InferenceSession(model_onnx.SerializeToString())
+        got = sess.run(None, {'input': X})[1]
+        dec = model.decision_function(X)
+        assert_almost_equal(got[:, 1], dec, decimal=4)
+        assert_almost_equal(-got[:, 0], dec, decimal=4)
```

Comment:
https://github.com/onnx/sklearn-onnx/issues/370 is not for binary classification. If you do this with 3 classes, you'll see missing -ve sign in raw_scores of decision_function().

Reply:
I did it with multiclass, it seems to work. I modified the unit test the same way I did for binary.